mining_test.go

```diff
@@ -45,7 +45,7 @@ func TestTxFeePrioHeap(t *testing.T) {
 		txpi, ok := prioItem.(*txPrioItem)
 		if ok {
 			if last.feePerKB < txpi.feePerKB &&
```

Comment:
Shouldn't this be:

```if txpi.feePerKB > last.feePerKB && .... {```

Otherwise it would incorrectly fail when the fees are the same which is perfectly acceptable.